extensions/exposition/source/directives/octets/Store.ts

```diff
@@ -141,12 +152,29 @@ export class Store implements Directive {
 
     return await this.discovery[key]
   }
+
+  private parseMeta (value: string | string[]): Record<string, string> {
+    if (Array.isArray(value))
+      value = value.join(',')
+
+    const meta: Record<string, string> = {}
```

Comment:
NIT: I'm seeing this pattern (creating empty object and then populating it's properties) very frequently in Toa codebase - is there a reason why we avoiding `Object.fromEntries(something.map(...)))` ?

Reply:
Not sure, just seems more readable to me.

Why `fromEntries` is better? In terms of performance and GC, it's not clear to me which is worse: update the object or create a bunch of arrays.

Reply:
It's definitely not better from V8 performance point of view (as many other higher level operations like `for(const a of iterable)` are still slower than `for(let i=0; i++; i<`) but it's more idiomatic tool of the language as what you actually want to here is to create an object from entries and [Object.fromEntries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) is the way the language runtime provides for that.

Reply:
Each time I write JavaScript that creates an object it hurts me a little bit. Constructing an array of tuples within a loop is simply a lot of pain.